source/class/qx/tool/compiler/ClassFile.js

```diff
@@ -2896,6 +2922,18 @@ qx.Class.define("qx.tool.compiler.ClassFile", {
       return this.__requiredAssets;
     },
 
+    /**
+     * Adds a required font
+     * @param {String} name  name of the font
```

Comment:
Is duplicating name fine? Noticed it in several places of PR.

Reply:
The second `name` could be capitalized to make it more clear.

I tend to format my JSDoc with the description on a new line, so that problem doesn't show up. I would have done this:
```
@param {String} name
    name of the font
```

Reply:
Got it, thank you